exercises/practice/rest-api/src/RestApi.elm

```diff
@@ -0,0 +1,36 @@
+module RestApi exposing (buildDatabase, get, post)
+
+import Dict exposing (Dict)
+import Json.Decode exposing (Error)
+import Json.Encode
+
+
+type alias Name =
+    String
+
+
+type alias User =
+    { name : Name
+    , owes : Dict Name Int
+    , owedBy : Dict Name Int
+    , balance : Int
+    }
+
+
+type alias Database =
+    Dict Name User
+
+
+buildDatabase : String -> Result Error Database
+buildDatabase payload =
+    Debug.todo "Please implement buildDatabase"
+
+
+get : Database -> String -> Maybe String -> String
```

Comment:
I think a similar thing applies here. I don't think its totally clear that the return value should be a json string encoding the "Response with payload" from the instructions.

Reply:
Maybe I could require that they return a `Json.Encode.Value`? 
Or more simply we could have a `type alias JsonString = String`.

Reply:
Yeah something like that sounds good. I think we also need to state what the format of the json should be, there are two options in the table in the instructions, and we want the one from the "Response with payload" one. Maybe we could even define a type alias for it, then ask for a function to encode that, and then ask to compose that function here? I'm not quite sure on the details ...

Reply:
I added the alias.
I thought that `Maybe JsonString` encapsulated well enough that the payload can missing, so I left it this way.